english/add_music.md

```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```

Comment:
This may be a stupid question - but how do you first create a directory on github?  I don't recall Caroline and Sarah saying how they did that - perhaps I missed it - thanks. ;-)